main.nf

```diff
@@ -78,19 +98,13 @@ workflow {
     }
 }
 
-workflow.onComplete {
-    sendMail( 
-        to: params.email,
-        subject: "Success: ${params.experimentName} succeeded.",
-        body: "Experiment run ${params.experimentName} using Encyclopedia succeeded.",
-    )
+
+// A dummy workflow for testing:
+workflow dummy {
+    channel.of('This workflow doesn\'t do anything...')
 }
```

Comment:
Do we have tests that utilize this function? If not, some should be written.

Reply:
Good point! I was using this workflow just to trigger email messages. I might should just delete it.